### DISCUSSION TITLE
Excel changing ROW to @ROW

### DISCUSSION THREAD

**Discussion by MarianiVentura** [dfinke/ImportExcel#1555] [General]
Good day.

I am wondering if anyone else is having the issue where Excel changes my ROW calculations to @ROW?

Consider the following code.  I am importing a csv, then creating the formula with ROW, but the final Excel changes this to @ROW.

I have manually do a search/replace in Excel to fix these or else the formula doesn't work.

Any ideas?

Thanks.

```markdown
 
    $excel = Import-Csv $outputFile -Delimiter ^ | Export-Excel $excelFile -Title $title -WorksheetName 'SalesOrders' -AutoSize -AutoNameRange -TableName SO -FreezePane 3 -Passthru

    $formula = "AVERAGE(IF(SUBTOTAL(9,OFFSET(`$J`$3:`$J`$$Frow,ROW(`$J`$3:`$J`$$Frow)-MIN(ROW(`$J`$3:`$J`$$Frow)),,1))>0,`$J`$3:`$J`$$Frow))"
    # Set the Excel range with the formula
    Set-ExcelRange -Range $sheet.Cells["J1"] -Formula $formula -BackgroundColor yellow

**Comment by dfinke** [comment]
@MarianiVentura have not seen that before. Please could you post some scrubbed csv data.

**Reply by MarianiVentura** [reply]
[SalesOrdersSummaryStory1342_20240112_FINKE.xlsx](https://github.com/dfinke/ImportExcel/files/13922319/SalesOrdersSummaryStory1342_20240112_FINKE.xlsx)

Attached is example file with just a few rows and customer names removed.  If you look at L1, you can see how it changed.

Thank you!

Here is the Powershell code:
`$formula = "AVERAGE(IF(SUBTOTAL(9,OFFSET(`$L`$3:`$L`$$Frow,ROW(`$L`$3:`$L`$$Frow)-MIN(ROW(`$L`$3:`$L`$$Frow)),,1))>0,`$L`$3:`$L`$$Frow))"`
